backend/otr/src/main/java/fi/oph/otr/onr/dto/ContactDetailsType.java

```diff
@@ -0,0 +1,31 @@
+package fi.oph.otr.onr.dto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public enum ContactDetailsType {
+  @JsonProperty("YHTEYSTIETO_SAHKOPOSTI")
+  EMAIL,
+
+  @JsonProperty("YHTEYSTIETO_PUHELINNUMERO")
+  PHONE_NUMBER,
+
+  // Currently unused, exists for json mapping
+  @JsonProperty("YHTEYSTIETO_MATKAPUHELINNUMERO")
+  MOBILE_PHONE_NUMBER,
+
+  @JsonProperty("YHTEYSTIETO_KATUOSOITE")
+  STREET,
+
+  @JsonProperty("YHTEYSTIETO_POSTINUMERO")
+  POSTAL_CODE,
+
+  @JsonProperty("YHTEYSTIETO_KUNTA")
+  TOWN,
```

Comment:
Meneekö kunta ja kaupunki nyt oikein? Kunta on municipality, ja sitäkö ONR käyttää osoitteen kaupunkina?

Reply:
Tsekkasin tätä sade-ympäristön kannasta ja oikeustulkkirekisterin tallettaman datan osalta on se ja sama kumpaa käytetään, meillä on nimittäin duplikoituna siellä kannassa kaikkien osalta sama arvo kuntana ja kaupunkina. Ja sama näkyy pätevän myös puhelinnumeron ja matkapuhelinnumeron välillä. Jos tuota tuotantodataa haluaa joskus siistiä, niin nuo duplikaattirivit voisi ehkä poistaa vaikka varmaan parempi olla tieten tahtoen onr:n sade-ympäristön kantaan koskematta.

Muutoin tosin "kaupunki" tyyppi yleisesti käytetty ja "kunta" tyypillä merkittävästi vähemmän rivejä kannassa. Näin ollen vtj-yhteystiedot pitäisi varmaan hakea sen alta. Vaihdankin käyttämään tätä kaupunkia.